src/js/messaging/components/MessageDetails.jsx

```diff
@@ -76,35 +76,34 @@ class MessageDetails extends React.Component {
       </div>
     );
 
-    const inputId = `message-details-${messageId}`;
-    const compactInputId = `compact-${inputId}`;
-
     const compactSentDate = sentDate && (
       <span>{formattedDate(sentDate, { fromNow: true })}</span>
     );
 
     return (
       <div
-          ref="detailsControl"
           className="messaging-message-details-control"
           tabIndex="-1"
-          onBlur={this.hideDetails}
+          onBlur={() => {
+            if (this.state.expandedFull) {
```

Comment:
Yup I think I've run into this situation before: it's an action that is used as a toggle in one case, but also reused to solely open/close in another case..

Really minor issue thought, this PR LGTM as is